lib/harvesting/client.rb

```diff
@@ -21,16 +20,18 @@ def initialize(access_token: ENV['HARVEST_ACCESS_TOKEN'], account_id: ENV['HARVE
       @access_token = access_token.to_s
       @account_id = account_id.to_s
 
-      if @account_id.length == 0 || @access_token.length == 0
-        raise ArgumentError.new("Access token and account id are required. Access token: '#{@access_token}'. Account ID: '#{@account_id}'.")
-      end
+      return unless @account_id.empty? || @access_token.empty?
```

Comment:
I feel like the new code is harder to reason about.

I would prefer:
``` ruby
if @account_id.length.emtpy? || @access_token.empty?
  raise ArgumentError.new("Access token and account id are required. Access token: '#{@access_token}'. Account ID: '#{@account_id}'.")
end
```

What you think?

Reply:
Yeah obviously!!!

we can configure it in the CI, I'd love to do that. I have more experience I Github Actions or CircleCI, If you want I can set one of them up to:

build, test, run rubocop

Reply:
The Travis config for this project is quite simple so I don't have a problem to change to another CI. Can you open an issue for about this and explain what you have in mind?